### ISSUE TITLE
Create some "getting started" material for newcomers to Jupyter

### ISSUE THREAD

**Issue by choldgraf** [2i2c-org/infrastructure#1415]
### Context

There are many cases where 2i2c hubs are used by newcomers to the Jupyter / PyData / etc ecosystem. Sometimes those newcomers aren't in an environment where they get a lot of introductory material from the instructors (e.g. as part of a hackweek). In these cases, it would help them make rapid progress if they had some high-quality notebooks / instructions / etc that they could follow in order to learn quickly on a hub.

ref: this was also [noted in the Jack Eddy event](https://github.com/2i2c-org/infrastructure/issues/1360#issuecomment-1154494181):

> A hub that starts with a directory of simple tutorials (and a minimal kernel that satisfies the needs of those tutorials) would be a good default "base hub" feature. Additional hub areas and custom kernels could be created depending on the subject/workshop area.

### Proposal

We should define a minimal set of "getting started" notebooks that work with the "base JupyterHub environment", and an easy way for hub users to grab copies of those notebooks themselves in order to learn more what they can do with Jupyter / JupyterLab / etc.

**Comment by dan800** [comment]
@fperez created two intro folders for the Jack Eddy event here: https://github.com/jack-eddy-symposium/intro-git-jupyter
One for git and one for jupyter. They would be great starter notebooks. As an aside, I'd also suggest that sorting authentication ahead of future events (e.g., by git id or orcid id) would help with getting the most out of these hubs. I think this could be handled part of event registration. @fperez also created a small notebook that authenticated from the hub to git that made life a lot easier during the event.

**Comment by BrainonSilicon** [comment]
Hi 2i2c team! I'm jumping in to say that I'd love to collab and share what our Community Management team is currently gathering together as part of some GitHub onboarding (I'm one of the Community Managers at The Alan Turing Institute).

**Comment by jmunroe** [comment]
I agree that there should be some "getting started" notebooks that try and ramp up users ("60 seconds to Jupyter", "Your first 5 minutes in Python", "Quickstart to using Python and Jupyter", or similar kind of thing) that are available for users as soon as they log in to a hub.  This documentation does not need to be comprehensive but could point to additional material for further information.  The goal could be to ensure users feel like they know just enough to able to participate in the hackweek / workshop / course / community.

I know that there are many resources already out there aimed at intro to python, pydata, jupyter, visualization, git, etc. We should try and pull content from those "upstream" sources where possible.  If the Turing Institute has such materials that would be awesome, but I'll note that [Project Pythia's Foundations](https://foundations.projectpythia.org/) book may be a good source.
